doc/api/assert.md

```diff
@@ -26,12 +29,42 @@ When using the `strict mode`, any `assert` function will use the equality used i
 the strict function mode. So [`assert.deepEqual()`][] will, for example, work the
 same as [`assert.deepStrictEqual()`][].
 
+Besides that error messages that involve objects produce an error diff instead of
```

Comment:
grammar nit: comma after the first `that`? Or maybe just start with `Error messages which involve objects …`?